remote/bitbucket/convert.go

```diff
@@ -186,5 +187,18 @@ func convertPushHook(hook *internal.PushHook, change *internal.Change) *model.Bu
 		build.Event = model.EventPush
 		build.Ref = fmt.Sprintf("refs/heads/%s", change.New.Name)
 	}
+	if len(change.New.Target.Author.Raw) != 0 {
+		build.Email = extractEmail(change.New.Target.Author.Raw)
+	}
 	return build
 }
+
+// extracts the email from a git commit author string
+func extractEmail(gitauthor string) (author string) {
+    re := regexp.MustCompile("<(.*)>")
```

Comment:
you can just put it right above the function. Please rename to `reEmail` so that we can differentiate from other global regexs, should we add them in the future. thanks :)

```diff
+var reEmail = regexp.MustCompile("<(.*)>")

// extracts the email from a git commit author string
func extractEmail(gitauthor string) (author string) {
-  re := regexp.MustCompile("<(.*)>")
    matches := reEmail.FindAllStringSubmatch(gitauthor,-1)
    if len(matches) == 1 {
        author = matches[0][1]
    }
    return
}
```